cleanlab/rank.py

```diff
@@ -28,18 +28,23 @@
 
 
 import numpy as np
+from typing import List
+import warnings
+from cleanlab.utils.label_quality_utils import subtract_confident_thresholds
 
 
 def order_label_issues(
-    label_issues_mask,
-    labels,
-    pred_probs,
+    label_issues_mask: np.array,
+    labels: np.array,
+    pred_probs: np.array,
```

Comment:
Not your responsibility, but since you already made the change for the other functions, could you also update docstring for these args here (in `order_label_issues` docstring) to what you did for the other methods:
```
labels : np.array
      Labels in the same format expected by the `score_label_quality()` method.
    pred_probs : np.array (shape (N, K))
      Predicted-probabilities in the same format expected by the `score_label_quality()` method.
```

Reply:
Yeah definitely. Done, thanks 
3dbbe7054aeae25b8bddd73a53838cce35b06fa9